src/compliment/sources/class_members.clj

```diff
@@ -80,9 +86,24 @@
   (get-in @members-cache [ns :members]))
 
 (defn class-member-symbol?
-  "Tests if a symbol name looks like a non-static class member."
-  [^String x]
-  (.startsWith x "."))
+  "Tests if `x` looks like a non-static class member,
+  e.g. \".getMonth\" or (with `clj-1-12+` option set) \"java.util.Date/.getMonth\".
+
+  When true, yields `[klass-name method-name]`."
+  ([x] (class-member-symbol? x nil))
+  ([^String x {:keys [clj-1-12+]}]
```

Comment:
No need to make it depend on an explicit argument. The environment is either Clojure 1.12+ or it is not.